receiver/tlscheckreceiver/scraper_test.go

```diff
@@ -26,6 +26,7 @@ func mockGetConnectionStateValid(endpoint string) (tls.ConnectionState, error) {
 		NotAfter:  time.Now().Add(24 * time.Hour),
 		Subject:   pkix.Name{CommonName: "valid.com"},
 		Issuer:    pkix.Name{CommonName: "ValidIssuer"},
+		DNSNames:  []string{"example.com", "example.com/foo"},
```

Comment:
I don't think paths are valid in the SAN field. Perhaps we can change to `api.example.com` instead. Should we also test for wildcards (`*.example.com`)?

Reply:
Sure, why not. We're just reporting whatever the cert says. I have changed the fixture and the test to reflect.